docs/sources/release-notes/v2-9.md

```diff
@@ -32,14 +32,37 @@ Grafana Labs is excited to announce the release of Loki 2.9.0 Here's a summary o
   - The `-ingester.unordered-writes` CLI flag is deprecated and will always default to `true` in the next major release.
   - For the full list of deprecations, see CHANGELOG.md
 
-
 ## Bug fixes
 
+### 2.9.5 (2024-02-28)
+
+* Action repo reference ([#11807](https://github.com/grafana/loki/issues/11807)) ([fb02797](https://github.com/grafana/loki/commit/fb02797a355e0d264d4cbd46045bd3ff2ab121d7)).
+* Bump base images and Go dependencies to address CVEs ([#12092](https://github.com/grafana/loki/issues/12092)) ([eee3598](https://github.com/grafana/loki/commit/eee35983f38fe04543b169ffa8ece76c23c4217b)).

```

Comment:
those two are being listed as bug fixes, but they don't seem to be :thinking:

Reply:
I'll move them.  I threw this together really quickly at the end of the day.